### PR TITLE
Refactor the code to lower the indentation level

### PR DESCRIPTION
**Pull request by vladmos**

### CODE REVIEW

File: warn/warn_bazel.go [hunk 1]
```diff
@@ -89,23 +89,29 @@ func globalVariableUsageCheck(f *build.File, category, global, alternative strin
...
+		if ident.Name != global {
+			return
+		}
+		if binding := env.Get(ident.Name); binding != nil {
+			return
+		}
+		if fix {
+			// It may be not correct to just replace the ident's name with `alternative` as it may be something complex
+			// like `native.package_name()` which is not a valid ident, but it's fine for reformatting.
+			ident.Name = alternative
```
Comment (pmbethe09) [inline warn/warn_bazel.go]: opt: could return here, avoid the else {}

File: warn/warn_bazel.go [hunk 2]
```diff
@@ -248,26 +254,26 @@ func attrSingleFileWarning(f *build.File, fix bool) []*Finding {
...
 			start, end := singleFileParam.Span()
 			findings = append(findings,
 				makeFinding(f, start, end, "attr-single-file",
 					"single_file is deprecated in favor of allow_single_file.", true, nil))
+			return
+		}
+		value := singleFileParam.Y
+		if boolean, ok := value.(*build.Ident); ok && boolean.Name == "False" {
+			// if the value is `False`, just remove the whole parameter
+			call.List = append(call.List[:i], call.List[i+1:]...)
```
Comment (pmbethe09) [inline warn/warn_bazel.go]: same

File: warn/warn_bazel.go (hunk 1 above)
Comment (vladmos) [reply warn/warn_bazel.go]: Done.

File: warn/warn_bazel.go (hunk 2 above)
Comment (vladmos) [reply warn/warn_bazel.go]: Done.

**Comment by vladmos** [comment]
@pmbethe09 please mark the PR as approved, otherwise I can't merge it. Thanks

**Comment by vladmos** [comment]
> I thought I did approve it.

I think you did but I added another commit to address your comments which marked the PR as not approved. Thanks!

**Pull request merged**